src/renderer/pages/Project.tsx

```diff
@@ -1,28 +1,80 @@
-import { Stack } from '@mui/material';
-import { Box } from '@mui/system';
+import { Box, Stack } from '@mui/material';
+import { useCallback, useEffect, useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import StandardButton from 'renderer/components/StandardButton';
-import { projectOpened } from 'renderer/store/actions';
+import TranscriptionBlock from 'renderer/components/TranscriptionBlock';
+import { transcriptionCreated } from 'renderer/store/actions';
+import { Transcription } from 'sharedTypes';
 import { ApplicationStore } from '../store/helpers';
 
+/*
+This is the page that gets displayed while you are editing a video.
+It will be primarily composed of the transcription area, an editable text box whose
+changes get reflected in the video. In addition to that, there is a video preview
+section to the side among other things.
+*/
 const ProjectPage = () => {
+  const dispatch = useDispatch();
   const currentProject = useSelector(
     (store: ApplicationStore) => store.currentProject
   );
 
-  const dispatch = useDispatch();
+  // RK: I really shouldn't use transcriptionCreated here - but i'm lazy and it works
```

Comment:
Yeah, it works for now but it will start doing funky stuff if we change it (e.g. for the original transcription to be backed up into a different part of the state, which is likely at some point).

easy solution - make a new action that does the same thing for now, just call it `transcriptionEdited` or similar

harder (but much better) solution - refactor to use ops so the undo works and so you're not mutating the application state

Reply:
Will need to go with the harder option for undo anyway. Ok to just leave it as is for now?

Reply:
Hahah yeah leave it as is, I'd say merge this when the other minor stuff is resolved then make a new PR to switch to using the op system